src/main/java/org/zephyrsoft/sdb2/gui/MainWindow.java

```diff
@@ -1606,16 +1626,7 @@ public void focusLost(FocusEvent e) {
 		panelEdit.add(lblAdditionalCopyrightNotes, gbcLblAdditionalCopyrightNotes);
 		
 		comboBoxLanguage = new JComboBox<>();
-		comboBoxLanguage.addFocusListener(new FocusAdapter() {
-			@Override
-			public void focusLost(FocusEvent e) {
-				try {
-					handleSongDataFocusLost();
-				} catch (Throwable ex) {
-					handleError(ex);
-				}
-			}
-		});
+		comboBoxLanguage.setEditable(true);
```

Comment:
Won't this (in combination with storing it as string instead of as enum) lead to chaos in the entered languages? Some might enter it with capital, some without, some might abbreviate it...

Reply:
Hmm..
Then i would suggest either: Storing the languages as iso codes and listing all (JVM-Supported) Languages-Names (of the current Locale) inside the Dropdown.
```
Set<String> languages = Arrays.stream(Locale.getISOLanguages())
            .map(Locale::new)
            .map(Locale::getDisplayLanguage)
            .collect(Collectors.toCollection(TreeSet::new));
```
![grafik](https://user-images.githubusercontent.com/17047770/121131206-ba53bc00-c82f-11eb-94de-d8420481eb00.png)
We could add all used Languages to the top, to prevent from scrolling.

Or leaving the Combo editable and listing already used languages as help:
![grafik](https://user-images.githubusercontent.com/17047770/121132556-92fdee80-c831-11eb-9064-f3cecda05049.png)
In that way, the user can enter multiple languages together.

Reply:
Ok, what do you think about the second solution? (User can choose of already used languages or define a new one)

This would not break current installations, and gives the user more freedom.

Reply:
The second solution sounds fine to me.